modules/ROOT/nav.adoc

```diff
@@ -1,11 +1,12 @@
 .xref:index.adoc[Anypoint Platform]
-* xref:index.adoc[MuleSoft Documentation]
+* xref:index.adoc[Documentation]
```

Comment:
Just making the left nav simpler. The actual file topic is still MuleSoft Documentation, though it probably should be Anypoint Platform Documentation for accuracy--I think people search MuleSoft more than AP.

Reply:
Would you be willing to craft an entry for our Style Reference about this?
"Anypoint Platform" vs. "MuleSoft"?

Reply:
Absolutely. will follow up in slack.